README.md

```diff
@@ -36,7 +36,7 @@ Iter-tools is at present the only fully-featured library of its kind. Here is wh
 - **Pay what you need on web**: Iter-tools is designed to be modular, meaning that only the methods you actually use end up being bundled and shipped if you are using a supported bundler. Webpack/uglify are capable of tree-shaking iter-tools, as is Rollup after v1.3.0. If you are not using a bundler that supports tree-shaking, you can still transpile an expression like `import { map, filter } from "iter-tools"` into imports of individual files: `iter-tools/es5/map` and `iter-tools/es5/filter`. To do this use the `iter-tools-explode-imports` package.
 
 ## Definitions
-This should help clarify the documentation. You can also get more informations here: https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Iterators_and_Generators and here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
```

Comment:
Why are you deleting these documentation links? Those are definitely supposed to be there

Reply:
That was a mistake my bad :( 

Sorry I wasn't paying attention...